examples/timeline-charts-example/src/Dashboard.jsx

```diff
@@ -0,0 +1,202 @@
+import React, { useRef, useCallback, useState } from "react";
+
+import { withStyles } from "@material-ui/core/styles";
+import Slider from "@material-ui/core/Slider";
+import ChartsEmbedSDK from "@mongodb-js/charts-embed-dom";
+import { useEffect } from "react";
+
+const firstOlympicsYear = 1896;
+const lastOlympicsYear = 2016;
+const timelineInterval = 2000; //ms
+
+let timerId;
+let replay = false;
```

Comment:
These variables should be defined and referenced inside the `Dashboard` component via refs
e.g.
```
const timerIdRef = React.useRef();
const replayRef = React.useRef(false);

...
timerIdRef.current = setInterval(() => {})
clearInterval(timerIdRef.current);
...
replayRef.current = true;
```

Reply:
Good idea, thanks :)